tests/test_sampling.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import pymc4 as pm
+import numpy as np
+from pymc4 import distributions as dist
+import tensorflow as tf
+
+
+@pytest.fixture(scope="function")
+def simple_model():
+    @pm.model()
+    def simple_model():
+        norm = yield pm.Normal("norm", 0, 1)
+        return norm
+
+    return simple_model
+
+
+@pytest.fixture(scope="function")
+def simple_model_with_deterministic(simple_model):
+    @pm.model()
+    def simple_model_with_deterministic():
+        norm = yield simple_model()
+        determ = yield pm.Deterministic("determ", norm * 2)
+        return determ
+
+    return simple_model_with_deterministic
+
+
+@pytest.fixture(scope="module", params=[True, False], ids=str)
```

Comment:
Better to use descriptive ids xla/noxla. True/False are out of context in test log.